sysutils/broot/Portfile

```diff
@@ -20,12 +20,14 @@ long_description    broot is a new way to see and navigate directory trees. \
                     via regular expressions, and more.
 
 checksums           ${distname}${extract.suffix} \
-                    rmd160  71728d0ed1a584c86bb937424a31ca556596a0fb \
-                    sha256  85cc6ed343f5e9df28b54c5e842de1ca8351eebe384bea1faa0e54483707bc4d \
-                    size    1681528
+                    rmd160  fd504f540c4a2fbd574a831b8cbffd93c4087115 \
+                    sha256  aa0ecf6f5a25ac6af027ee627118f323467b9fe0f36fba5a3086219fbdceef1a \
+                    size    2090581
 
 destroot {
     xinstall -m 755 ${worksrcpath}/target/[cargo.rust_platform]/release/${name} ${destroot}${prefix}/bin/
+    xinstall -m 644 ${worksrcpath}/man/page ${destroot}${prefix}/share/man/man1/broot.1
+    system   -W "${destroot}${prefix}/share/man/man1" "gzip ./broot.1"
```

Comment:
You should remove the `system` line that compresses the manpage. MacPorts will compress manpages automatically.

Reply:
OK, thank you @ryandesign - change made.